scripts/energywind.lua

```diff
@@ -15,12 +15,13 @@ local turnSpeed = math.rad(20)
 local isWind, baseWind, rangeWind
 
 local function BobTidal()
-	baseDirection = baseDirection + math.random(0,math.rad(2))
+	local tideDirection = 0
 	while true do
-		Turn(cradle, y_axis, baseDirection, math.rad(1))
+		tideDirection = tideDirection + math.random() - 0.5 * math.rad(2)
```

Comment:
Right now this line increments the direction value by between 0.0 and 1.0 radians (i.e. up to 57ish degrees with an average of 28ish) and then decrements by 1 degree (i.e. half of two degrees) for an average change of 27ish. This is much faster than the turn speed of 1 degree/s can keep up (the direction target runs circles around the turbine).

I believe this was supposed to be
```suggestion
		tideDirection = tideDirection + (math.random() - 0.5) * math.rad(2)
```
On the other hand not keeping up looks marginally better in that it keeps swaying while the "fixed" version will turn (for example) 0.3 degrees in 300ms, then stop for the remaining 700ms of a cycle. Perhaps save the random roll in a variable and scale the speed accordingly as well?

Reply:
Thanks for the find.